docs/setup.md

```diff
@@ -43,7 +43,7 @@ FetchError: request to https://idapi.thegulocal.com/auth?format=cookies failed,
 
 ### Environment File
 
-Populate `.env` and `cypress-ete.env` files by using the examples from [`.env.example`](../.env.example) and [`cypress-ete.env.example`](../cypress-ete.example.env). These `.env` files should **never** be committed as they store secrets.
+Populate `.env` and `cypress-ete.env` files by using the examples from [`.env.example`](../.env.example) and [`cypress-ete.env.example`](https://github.com/guardian/gateway/blob/main/cypress-ete.env.example). These `.env` files should **never** be committed as they store secrets.
```

Comment:
Does this need to have the full URL? And does it make sense to have the full url for `cypress-ete.env.example` but not `.env.example`?

Also... I thought I fixed this link very recently...